### ISSUE TITLE
[camera]: worked before, now only shows dark background

### ISSUE THREAD

**Issue by star-buck** [blue-systems/pm#197]

**Comment by bhush9** [comment]
Same gstreamer mess like issue #201 

I think we need to build qt5 with gstreamer 1.0

**Comment by bhush9** [comment]
Now one no-longer gets gstreamer errors like that but now instead rainbow bars are shown... meh..

**Comment by star-buck** [comment]
img13: now doesnt even start anymore

**Comment by bhush9** [comment]
Yeah, camera is known to break.. our newer libhybris is incompatible with qtubuntu-camera plugin for qtmultimedia. I am working on solution.. most probably it will be fixed in next image or image+1.

**Comment by star-buck** [comment]
camera app shows colored standstill image, but doesnt seem to take pictures.

**Comment by bhush9** [comment]
I at one point started qtmultimedia plugin for the camera which used android HAL api's.. however I didn't make much progress, mainly because I don't fully understand android API etc. I've one possible solution which is compiling compat module of libhybris with CM image.. and use qtubuntu-camera from ubuntu.

Though we wouldn't want to compile in compat module as it would conflict with input in kwin_wayland.. Leaving no choice but broken camera for now.

**Comment by bhush9** [comment]
This will need the gst-droid packaged. I will include it in next image, I already have it working locally.

**Comment by bhush9** [comment]
Update: Everything needed is packaged, but needs to be bundled inside the image. However there is a conflict between the codecs installed in ticket #277 as mentioned there. There is option to solve it which is gst-omx, and I've it packaged also, most of the formats are also working there with hardware accelerated video.. :-)

However it is possible that my testing is limited, so i haven't shipped it in image yet, @star-buck can you tell me which videos you have tested, so I can test it with newer setup and fix where fixes are needed?

So far in my testing following works:

- Big buck bunny 1080P, mp4, 30 and also 60 FPS
- Sintel trailer, mkv, 1080P (? FPS)
- Youtube: Star wars, Last jedi trailer (HD)
- Some more random youtube trailers like daredevil, iron-man etc..

And following doesn't work:

- Tears of steel 1080P, WebM, 60 FPS (it loads , but drops frames as movie progresses, however 1080P, MP4 works)

**Comment by bhush9** [comment]
@star-buck is camera required for demo at Dec 27? If so I can roll a image with camera fix..

**Comment by star-buck** [comment]
Nice to have yes, but not really needed.

**Comment by star-buck** [comment]
@bhush9 : please ship fix.

**Comment by bhush9** [comment]
`20180209-155413` fixes this
